pytorch_lightning/trainer/trainer.py

```diff
@@ -1538,7 +1538,7 @@ def _call_accelerator_hook(
         **kwargs: Any,
     ) -> Optional[Any]:
         self.lightning_module._current_fx_name = hook_name
-        fn = getattr(self.accelerator, hook_name)
+        fn = getattr(self.training_type_plugin, hook_name)
```

Comment:
why make this change? why not just call `_call_ttp_hook` instead of `_call_accelerator_hook`?

Reply:
Hey @awaelchli,

A small idea for the `test_fx_validator_integration` failure. The refactor didn't properly reset the hook name on the LightningModule as there are hooks that are nested such as the `training_step` with manual optimization. I think @carmocca has a draft PR to restore this behavior. Not sure if it could be the reason.

Reply:
This is fixed by #10575, but I don't have the time to finish it soon. If somebody wants to resolve the failing profiler tests, feel free to push to the PR.

Reply:
@carmocca  you mean #10957?